blenderproc/python/types/LightUtility.py

```diff
@@ -121,6 +121,7 @@ def setup_as_projector(self, pattern: np.ndarray, frame: Optional[int] = None):
         node_ox = nodes.get('Emission')
 
         image_data = bpy.data.images.new('pattern', width=pattern.shape[1], height=pattern.shape[0], alpha=True)
+        pattern = pattern / 255.0    # manual cast to range [0,1] to avoid integer casting issues below
```

Comment:
Thanks for the fix! We just need to make sure that this is not done if the given image is already a float image:
```suggestion
        if pattern.dtype == np.uint8:
            pattern = pattern / 255.0    # manual cast to range [0,1] to avoid integer casting issues below
```